modules/ptp-events-consumer-application.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp-cloud-events-consumer-dev-reference.adoc
+
+:_content-type: REFERENCE
+[id="ptp-events-consumer-application_{context}"]
+= PTP events consumer application reference
+
+PTP event consumer applications require the following features:
```

Comment:
I would remove the periods for the list items that follow since they aren't sentences.